enos/enos-samples-ce-build.hcl

```diff
@@ -9,6 +9,7 @@ sample "build_ce_linux_amd64_deb" {
       arch            = ["amd64"]
       artifact_source = ["crt"]
       artifact_type   = ["package"]
+      consul_edition  = ["ce"]
```

Comment:
As an outside observer, ["ce"] here is confusing with the below usage of `edition = ["ce"]`. This may be my lack of knowledge or understanding.

Reply:
`edition` refers to the vault edition; `consul_edition` refers to the consul edition. In this case, we're using just the community edition of each, because this sample (named `build_ce_linux_amd64_deb`) is building and testing only the community edition of Vault. Does that help?

Reply:
It does help, it feels like an overloaded term. If we are the only ones using it though it should be fine

Reply:
Are you saying that `edition` is overloaded or `ce`? Both are commonly used terms throughout the Vault/Enos codebase.